v2/manifests/deps/vault.yml

```diff
@@ -9,7 +9,7 @@ services:
       VAULT_DEV_ROOT_TOKEN_ID: '0000-0000-0000-0000'
       VAULT_DEV_LISTEN_ADDRESS: '0.0.0.0:8200'
     healthcheck:
-      test: [ "CMD", "wget", "-q", "-O", "-", "http://localhost:8200/v1/sys/health?drsecondarycode=200&performancestandbycode=200&standbycode=200" ]
+      test: [ "CMD", "wget", "-q", "-O", "-", "http://127.0.0.1:8200/v1/sys/health?drsecondarycode=200&performancestandbycode=200&standbycode=200" ]
       interval: 60s
```

Comment:
why didn't localhost work in this instance? (interested)

Reply:
My potentially less than helpful comment message:
> The vault health check was failing due to `localhost` not resolving for
`wget` in the container. Switching to avoid the resolution resolves the
issue.

I honestly don't know why though. The `/etc/hosts` had an entry for `localhost` resolving to `127.0.0.1` and one to `::1`. `wget` wouldn't resolve it though. As using `127.0.0.1` worked around the issue I decided not to invest the time in working out why, but I was also curious!

Reply:
weird. Ok!